integration-tests/tests/src/tests/sync/mongo-db-client.ts

```diff
@@ -0,0 +1,188 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2023 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+import { expect } from "chai";
+import { ChangeEvent, Document, InsertEvent, MongoDBCollection } from "realm";
+
+import { authenticateUserBefore, importAppBefore } from "../../hooks";
+import { sleep } from "../../utils/sleep";
+
+type TestDocument = {
+  _id: number;
+  text?: string;
+  isLast?: boolean;
+};
+
+describe.skipIf(environment.missingServer, "MongoDB Client", function () {
+  this.timeout(60_000); // TODO: Temporarily hardcoded until envs are set up.
```

Comment:
very nit this is more of a personal stylistic preference and I actually had no idea you could do this (seems like this was added in 2021 😮) but because the number is not large and this formatting is not used anywhere in the codebase else I'd prefer just `60000`

Reply:
Please don't make us count digits. If you *really* don't like `60_000`, then at least do `60*1000` which is the old way to write that before digit separators were allowed.

Reply:
yeah that's fair... I guess I just didn't like how `_` was chosen for this by the language(s) but then again `.` / `,` would have been a regional mess